lib/daru/io/exporters/excel.rb

```diff
@@ -23,34 +39,96 @@ class Excel < Base
         #
         #   Daru::IO::Exporters::Excel.new(df, "dataframe_df.xls").call
         #
-        # @todo The +opts+ parameter isn't used while creating the Excel Spreadsheet
-        #   yet. Implementing this feature will greatly allow the user to generate a
-        #   Spreadsheet of their choice.
+        # @example Writing to an Excel file with options
+        #   df = Daru::DataFrame.new([[1,2],[3,4]], order: [:a, :b])
+        #
+        #   #=> #<Daru::DataFrame(2x2)>
+        #   #      a   b
+        #   #  0   1   3
+        #   #  1   2   4
+        #
+        #   Daru::IO::Exporters::Excel.new(df,
+        #     "dataframe_df.xls",
+        #     formatting: {
+        #       header: { color: :red, weight: :bold },
+        #       index:  { color: :green },
+        #       data:   { color: :blue }
+        #     },
+        #     display: { index: false }
```

Comment:
Hmm, the use-case I was considering is when a daru-io user decides to fix their `:formatting` and just decide to play-around with the `:display` hash. 

BTW, after having gone through the daru-view repository during the review discussion, I came across some frequently-used default options being declared as a class variable like `Daru::View.adapter = :highcharts`. Thinking about it, could it be used to define some default options? (Maybe, coupled with meta-programming?) That is, something like `Daru::IO::Exporters::Excel.index = {color: :green}`? Or does this seem complex?